README.rst

```diff
@@ -56,10 +73,10 @@ Instructions For Installation
                 Size: Discrete 2592x1944
                         Interval: Discrete 0.067s (15.000 fps)
                 Size: Discrete 0x0
+
 3. You can choose resolution from here.
 
-4. Camera ID is mostly 0, but you can replace video0 with video1
-in above command.
+4. Camera ID is mostly 0, but you can replace video*0* with video*1* in above command.
```

Comment:
It should be "Above example uses Camera ID 0 to indicate use of /dev/video0 device, please use the applicable value for your setup."